sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -288,6 +288,18 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 			oci.WithHostNamespace(runtimespec.NetworkNamespace),
 			oci.WithHostHostsFile,
 			oci.WithHostResolvconf,
+			// Unmask `/sys/firmware` by passing an alternate list of masked paths
+			oci.WithMaskedPaths([]string{
+				"/proc/acpi",
+				"/proc/asound",
```

Comment:
Can you link to where you got this list? I'd like to know how we make sure it remains up-to-date or at least ensure we all think its correct.

Reply:
It's part of [`oci/spec.go`](https://github.com/containerd/containerd/blob/6906b57c721f9114377ceb069662b196876915c0/oci/spec.go#L164). I would like to implement something a little better where we fetch the `MaskedPaths` from the spec generated by `populateDefaultUnixSpec` and then just drop `/sys/firmware` from the list, but that may have to be implemented in a follow-up PR.

Reply:
That works for me.

Reply:
I'd recommend putting the link in a comment above the code, at a minimum.